package.json

```diff
@@ -19,6 +19,9 @@
     "number": 8467,
     "sha": "6cb7fec4e154faa0a4a3fee4b33dfef91b9870d9"
   },
+  "wazuh": {
+    "version": "4.4.0"
```

Comment:
Bump to 4.5.0

Tests and snapshots should use `4.x.x`, `x.y.z` or similar.

Reply:
Done